Tools/peg_generator/scripts/benchmark.py

```diff
@@ -41,9 +41,6 @@
     "compile", help="Benchmark parsing and compiling to bytecode"
 )
 command_parse = subcommands.add_parser("parse", help="Benchmark parsing and generating an ast.AST")
-command_check = subcommands.add_parser(
-    "check", help="Benchmark parsing and throwing the tree away"
-)
```

Comment:
I deleted the `make bench` target, since if we were to change it to something like `$(VENVPYTHON) scripts/benchmark.py --parser=pegen --target=stdlib parse`, it would be the same thing with `time_stdlib` (the new one).